pediatric_rootlets/pediatric_rootlets.sh

```diff
@@ -0,0 +1,148 @@
+#!/bin/bash
+#
+# This script performs:
+# - segmentation of rootlets from T2w data (model model-spinal-rootlets_ventral_D106_r20240523),
+# - segmentation of spinal cord from T2w data (sct_deepseg_sc)
+# - segmentation of vertebral levels from T2w data (sct_label_vertebrae)
+# - detection of PMJ from T2w data (sct_detect_pmj).
+#
+# Authors: Katerina Krejci, Jan Valosek
+#
+
+# Uncomment for full verbose
+set -x
+
+# Immediately exit if error
+set -e -o pipefail
+
+# Exit if user presses CTRL+C (Linux) or CMD+C (OSX)
+trap "echo Caught Keyboard Interrupt within script. Exiting now.; exit" INT
+
+# Retrieve input params
+SUBJECT=$1    # sub-0001/ses-01
+
+# get starting time:
+start=`date +%s`
+# FUNCTIONS
+# ==============================================================================
+
+# Segment spinal cord if it does not exist
+segment_if_does_not_exist(){
+  FILESEG="${SUBJECT}_${FILE_TYPE}_T2w_label-SC_mask.nii.gz"
```

Comment:
I believe there is no need to repeat `T2w` here. `file_t2` (L101 or L104) already contains `T2w`.
See [this line](https://github.com/spine-generic/spine-generic/blob/73e4aa9f5db653b8ef4eca8259dbefe6b0160bb4/process_data.sh#L92) to get inspiration on how to update the code.

Reply:
Resolved in [8be3c0a](https://github.com/ivadomed/model-spinal-rootlets/pull/53/commits/8be3c0a6f86331f6f29ac1d38eff028090046f7d).